### PR TITLE
Feature/storage

### PR DESCRIPTION
**Pull request by daudi13**
in this PR I:

- Added the main.rb file
- Added the App.rb file that will contain:
   - the welcome message
  - the selection menu
  - all methods responsible for listing, and creating items*
---
Thank you for taking the time to review my code.

* I had to subdivide the display_list method into two because it was't passing this Rubocop metric ` Metrics/CyclomaticComplexity: Cyclomatic complexity for display_list is too high. [8/7]`

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  def prompt
+    puts '
+    1 List all books
+    2 List all music albums
+    3 List all movies
+    4 List of games
+    5 List all genres
+    6 List all labels
+    7 List all authors
+    8 List all sources
```
Comment (iLynette) [inline app.rb:34]: Hello @daudi13 we do not have a fourth partner so we won't be needing sources

**Review by iLynette — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @daudi13 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

**Review by Tufahel — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @daudi13 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

File: app.rb (hunk 1 above)
Comment (daudi13) [reply app.rb:34]: noted I will remove it from the file. Thank you :+1:

**Pull request merged**